### ISSUE TITLE
Cannot debug with Office 365 

### ISSUE THREAD

**Issue by n07cn3** [cristianbuse/VBA-UserForm-MouseScroll#15]
Hello

Do you have any other report about this issue?

I just had Office 365 installed on my computer and now every time I try to debug the excel just close out.

This also happen with your demo file I downloaded.

Can you please fix this?

I really like this code.

Thank you,

Calvin

**Comment by cristianbuse** [comment]
@n07cn3 

Hi Calvin,

Thanks for raising the issue. Indeed something changed lately and the hook stopped working properly. You are the first to notice the issue.

 I just pushed a workaround to the repository. Could you please check if it works and solves your problem?

Cheers,
Cristian

**Comment by n07cn3** [comment]
Cristian,
thank you for looking into this.
i've download the latest copy and it also crashed while debug.

my work around for now is to disable the function while in debug and enable it back after i am done.

thanks again for looking into this.

Calvin

**Comment by cristianbuse** [comment]
@n07cn3 

Thanks Calvin!

It seems that the issue I found is a separate thing which had to be fixed anyway.

In the code module there is a note at the top:
> Do not debug code while the hook is active to avoid crashes!

I wrote that a long time ago, since the initial version. That's because I was never able to debug while on 64-bit Office. So, it seems you not only upgraded to Office 365 but also to 64-bit version of Office.

There is no way around the issue when using hooks. I would have to rewrite the code to use a timer or a loop to peek at mouse messages but I won't do that any time soon.

Thanks again!
